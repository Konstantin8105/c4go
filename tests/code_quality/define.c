#define A 1
#define B "hello world"
#define C a + 1

void t()
{
    int a = A;
    printf(B);
    int c = C;
    printf("%d", c);
}
