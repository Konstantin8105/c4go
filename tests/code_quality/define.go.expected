//
//	Package - transpiled by c4go
//
//	If you have found any issues, please raise an issue at:
//	https://github.com/Konstantin8105/c4go/
//

package code_quality

// t - transpiled function from  C4GO/tests/code_quality/define.c:5
func t() {
	var a int32 = 1
	printf([]byte("hello world\x00"))
	var c int32 = a + 1
	printf([]byte("%d\x00"), c)
}
